ci-operator/config/openshift/hypershift/openshift-hypershift-release-4.14__periodics-mce.yaml

```diff
@@ -128,6 +133,33 @@ tests:
     env:
       MCE_VERSION: "2.4"
     workflow: hypershift-mce-ibmz-conformance
+- as: e2e-aws-critical
+  cron: 0 0 29 2 *
+  steps:
+    cluster_profile: aws
+    dependencies:
+      HOSTEDCLUSTER_RELEASE_IMAGE_LATEST: release:414-latest
+    env:
+      MCE_VERSION: "2.4"
+      TEST_SUITE: openshift/conformance/parallel/minimal
```

Comment:
For my understanding. Could you please describe which tests are run as part of this "minimal" test suite? I see this: https://github.com/openshift/release/blob/master/ci-operator/step-registry/hypershift/conformance/hypershift-conformance-chain.yaml#L158 , which somehow reduces the scope but it's not very clear. The code suggests that it will exclude any tests with `[Suite:openshift/conformance/parallel]` but then, what tests actually remain the that group? Thanks

Reply:
https://github.com/openshift/release/blob/7017136ec27c157d9f6d9ce7ed7a3938289e2906/ci-operator/step-registry/hypershift/conformance/hypershift-conformance-chain.yaml#L158-L161

Reply:
https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_release/61897/rehearse-61897-periodic-ci-openshift-hypershift-release-4.17-periodics-mce-e2e-agent-critical/1893200060473151488
openshift/conformance/parallel/minimal will run about 1000 cases

Reply:
>  I think the following comments should make it easier for you to understand.

@LiangquanLi930 Thanks for the context.